docs/user/dk_bench.md

```diff
@@ -0,0 +1,185 @@
+  critic_lm = ChatOpenAI(model=judge_model_name, api_key=judge_openai_api_key)
+Evaluating: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 10/10 [00:06<00:00,  1.44it/s]
+INFO 2025-02-04 16:17:56,038 instructlab.model.backends.vllm:494: Waiting for GPU VRAM reclamation...█████████████████████████████████████████████████████████████████████████████████████████████████████████| 10/10 [00:06<00:00,  1.21it/s]
+
+# DK-BENCH REPORT
+
+## MODEL: granite-7b-lab
+
+Question #1:     5/5
+Question #2:     5/5
+Question #3:     5/5
+Question #4:     5/5
+Question #5:     2/5
+Question #6:     3/5
+Question #7:     2/5
+Question #8:     3/5
+Question #9:     5/5
+Question #10:     5/5
+----------------------------
+Average Score:   4.00/5
+Total Score:     40/50
+
+Responses and scores written to:
+~/.local/share/instructlab/internal/eval_data/dk_bench/granite-7b-lab/results_2025-02-04T16:18:02.337010.jsonl
+
+INFO 2025-02-04 16:18:02,337 instructlab.model.evaluate:176: ᕦ(òᴗóˇ)ᕤ Model Evaluation with DK-Bench completed! ᕦ(òᴗóˇ)ᕤ
+```
+
+Here is an example that uses flags to set the output file formats and the directory where the output files are written.
+
+```shell
+(venv) $ ilab model evaluate --benchmark dk_bench --input-questions questions.jsonl --model ~/.cache/instructlab/models/instructlab/granite-7b-lab --output-file-formats csv,xlsx --output-dir ~/dk-bench-results-dir
+INFO 2025-02-04 16:15:55,707 numexpr.utils:162: NumExpr defaulting to 16 threads.
+...
+Responses and scores written to:
+~/dk-bench-results-dir/granite-7b-lab/results_2025-02-04T16:18:02.337010.csv
+~/dk-bench-results-dir/granite-7b-lab/results_2025-02-04T16:18:02.337010.xlsx
+
+INFO 2025-02-04 16:18:02,337 instructlab.model.evaluate:176: ᕦ(òᴗóˇ)ᕤ Model Evaluation with DK-Bench completed! ᕦ(òᴗóˇ)ᕤ
+```
+
+## Configurations & Flags for Advanced Users
+
+There are also flags for temperature, system prompt, and a changing the judge model for users interested in seeing the difference those settings make to evaluation results.
+
+The `--temperature` flag sets the temperature for the model when it gets response for the questions. The default is `0.0`.
+The `--system-prompt` flag sets the system prompt for the model when it gets response for the questions. The default system prompt depends on the model architecture of the model being evaluated.
+The `--judge-model` flag sets the system prompt for the model when it gets response for the questions. The default is `gpt-4o`. The judge model needs to be an Open AI model.
+
+All of the flags for DK-Bench have variables that can be set in the config file as well.
+
+## Running DK-Bench with generated responses
+
+DK-Bench can be used with responses already provided in the `.jsonl` input file. If responses are already provided then no model needs to be provided as an argument to `ilab model evaluate`.
```

Comment:
Wording

```suggestion
If you've already generated the answers ahead of time, or would like to avoid spinning up a student model for this benchmark, you can also pass a `.jsonl` file containing the model's responses in the `response` field.
```